devops_toolset/project_types/wordpress/wptools.py

```diff
@@ -5,7 +5,7 @@
 import pathlib
```

Comment:
It is just a matter of keeping everything based on standards. No more Windows rules.